src/__screenshot_tests__/menu-screenshot-test.tsx

```diff
@@ -0,0 +1,35 @@
+import {openStoryPage, screen} from '../test-utils';
```

Comment:
Worth it to test a menu inside one of the components that were interacting with the visibility: hidden? So that any future changes to menu or data-card,etc.. will not have similar wrong interaction?

Reply:
Update: added a story and a test case for this.

Reply:
It's a good idea to test it here, yes